packages/saltcorn-cli/README.md

```diff
@@ -89,7 +89,7 @@ OPTIONS
   -f, --force  force command execution
 ```
 
-_See code: [src/commands/add-schema.js](https://github.com/saltcorn/saltcorn/blob/v0.9.6-beta.0/src/commands/add-schema.js)_
+_See code: [src/commands/add-schema.js](https://github.com/saltcorn/saltcorn/blob/v0.9.6-beta.0/packages/saltcorn-cli/src/commands/add-schema.js)_
```

Comment:
unfortunately these are created automatically on release by oclif and would be overwritten. I think we need to fix the ocliff configuration instead